board/tsat/3500/common/rootfs_overlay/etc/init.d/S90savelogs

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# Copy terminal logs to persistent storage on shutdown
+#
+
+LOG_DIR='/var/log'
+
+start() {
+	echo "Unpacking logs..."
+	# have to do this in two steps since busybox tar doesnt support gzip
+	gunzip -dc /root/logs.tar.gz | tar xf - -C $LOG_DIR
```

Comment:
I suggest putting "/root/logs.tar.gz" in an env-variable and reusing it in the stop() function